health-monitors/gpu-health-monitor/gpu_health_monitor/dcgm_watcher/dcgm.py

```
@@ -12,20 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import dcgm_structs, dcgm_errors, dcgm_fields, dcgmvalue, pydcgm, bisect
+import dcgm_agent, dcgm_structs, dcgm_errors, dcgm_fields, dcgmvalue, pydcgm, bisect
 import logging as log
 from . import types, metrics
 from gpu_health_monitor.metadata import MetadataReader
 from threading import Event
 from ctypes import *
```

Ruff (0.15.18)

[error] `from ctypes import *` used; unable to detect undefined names

(F403)